code/style.css

```diff
@@ -1,4 +1,327 @@
+* {
+  box-sizing: border-box;
+}
+
 html {
-  background: #0025ff;
-  color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
+  background: beige;
+  color: black;
+}
+
+.flex-parent {
+  background: darkslategray;
+  color: gold;
+  margin: 5px;
+  padding: 5px;
+  border: black solid 4px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 300px;
+}
+
+.flex-child {
+  margin: 5px;
+  padding: 10px;
+  width: 100%;
+  font-size: 20px;
+  display: flex;
+}
+
+.flex-logo {
+  display: none;
+}
+
+.flex-child1 {
+  display: none;
+}
+
+.big-news {
+  background: darkslategray;
+  margin: 5px;
+  padding: 10px;
+  border: black solid 4px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+  width: 100%;
+}
+
+.big-news-img {
+  margin: 5px;
+  width: 30%;
+}
+
+.shadow-1 {
+  width: 80%;
+  height: auto;
+  border-radius: 25px;
+  box-shadow: 0 0 20px 0 black;
+}
+
+.big-news-text {
+  margin: 5px;
+  width: 70%;
+  text-align: center;
+  font-size: 14px;
+}
+
+.big-news a:link,
+a:visited {
+  color: black;
+  text-decoration: none;
+}
+
+.big-news a:hover {
+  color: deeppink;
+}
+
+.shadow {
+  width: 80%;
+  height: auto;
+  border-radius: 25px;
+  box-shadow: 0 0 20px 0 black;
+}
+
+.grid-container {
+  background: darkslategray;
+  margin: 5px;
+  padding: 10px;
+  border: black solid 4px;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 10px;
+  width: 100%;
+  font-size: 20px;
+}
+
+.grid-item {
+  margin: 5px;
+  padding: 5px;
+  border-bottom: 2px solid black;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  width: 100%;
+  height: auto;
+}
+
+.grid-item a:link,
+a:visited {
+  color: black;
+  text-decoration: none;
+}
+
+.grid-item a:hover {
+  color: deeppink;
+}
+
+.footer {
+  background: darkslategray;
+  color: gold;
+  margin: 5px;
+  padding: 10px;
+  border: 4px solid black;
+  height: 200px;
+  width: 100%;
+  font-size: 16px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer-text {
+  margin: 5px;
+  padding: 10px;
+  width: 50%;
+}
+
+.footer-logo {
+  margin: 5px;
+  padding: 10px;
+  width: 50%;
+}
+
+.technigo-logo {
+  width: 150px;
+  height: 150px;
+}
+
+@media (min-width: 667px) and (max-width: 1024px) {
+  body {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .flex-parent {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
Since this element already got some styling outside of your media queries:
```
.flex-parent {
  background: darkslategray;
  color: gold;
  margin: 5px;
  padding: 5px;
  border: black solid 4px;
  display: flex;
  flex-direction: row;
  flex-wrap: nowrap;
  justify-content: center;
  align-items: center;
  width: 100%;
  height: 300px;
}
```
you only have to specify what is different in this media query:
```
.flex-parent {
  flex-wrap: nowrap;
}
```